ga4gh/server/repo/models.py

```diff
@@ -86,7 +79,6 @@ class Biosample(BaseModel):
     individualAgeAtCollection = pw.TextField(null=True)
 
     class Meta:
-        db_table = 'Biosample'
```

Comment:
They were an extra string that named the tables. Now they are named by the class name (less code same functionality). It does require you rebuild the test data since they may change.

The answer is no, there isn't another library that chooses the tables in our registries by name, but this may affect any ETL scripts that act on the registry by table name.